package.json

```diff
@@ -74,7 +74,7 @@
     "global": "^4.3.2",
     "invariant": "^2.2.4",
     "react": "^16.2.0",
-    "react-reconciler": "^0.7.0",
+    "react-reconciler": "0.10.0",
```

Comment:
Let's start using exact version numbers?

Reply:
what is the advantage to doing that?

Reply:
So that without our knowledge the wrong version doesn't get installed. Also, ~ and ^ assume semver is obeyed. Since react-reconciler is still experimental and constantly changing, it doesn't make sense to rely on ^ and ~

Reply:
sure, I was assuming tho that it followed the general node pattern of bumping the minor version for breaking changes, in which case either ~ or ^ is safe, since they will only ever bump when the last (patch) number changes)